chat-widget/src/components/livechatwidget/common/registerTelemetryLoggers.ts

```diff
@@ -25,13 +26,27 @@ export const registerTelemetryLoggers = (props: ILiveChatWidgetProps, dispatch:
         if (props.chatConfig) {
             telemetryData = TelemetryHelper.addChatConfigDataToTelemetry(props?.chatConfig, telemetryData);
         }
-        telemetryData = TelemetryHelper.addWidgetDataToTelemetry(telemetryConfig, telemetryData);
+
+        if (!props.chatSDK?.omnichannelConfig?.orgId || props.chatSDK?.omnichannelConfig?.orgId.trim().length === 0 ) {
+            throw new Error("orgId is undefined in ChatSDK");
+        }
+
+        if (!props.chatSDK?.omnichannelConfig?.widgetId || props.chatSDK?.omnichannelConfig?.widgetId.trim().length === 0 ) {
+            throw new Error("widgetId is undefined in ChatSDK");
+        }
+
+        if (!props.chatSDK?.omnichannelConfig?.orgUrl || props.chatSDK?.omnichannelConfig?.orgUrl.trim().length === 0  ) {
+            throw new Error("orgUrl is undefined in ChatSDK");
+        }
+
         telemetryData.OCChatSDKVersion = telemetryConfig.OCChatSDKVersion ?? "0.0.0-0";
         telemetryData.chatComponentVersion = telemetryConfig.chatComponentVersion ?? "0.0.0-0";
         telemetryData.chatWidgetVersion = telemetryConfig.chatWidgetVersion ?? "0.0.0-0";
         telemetryData.orgId = props.chatSDK?.omnichannelConfig?.orgId;
```

Comment:
It looks like `orgId`, `widgetId`, `orgUrl` is always retrieved from `ChatSDK`. Would it be possible to not expose these properties anymore?

Reply:
I'll create a task for it